### PR TITLE
Translate Mesecons to German

### PR DESCRIPTION
**Pull request by Wuzzy2**
This translate the Mesecons modpack into German.

### CODE REVIEW

File: mesecons_detector/locale/mesecons_detector.de.tr [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# textdomain: mesecons_detector
+Player Detector=Spielerdetektor
+Node Detector=Node-Detektor
```
Comment (numberZero) [inline mesecons_detector/locale/mesecons_detector.de.tr:3]: Shouldn’t that be “Block”? https://dev.minetest.net/Translating/de

**Comment by Wuzzy2** [comment]
Yes, right. I will fix it tomorrow (hopefully). I wait for more comments first.

**Comment by Wuzzy2** [comment]
Done.

File: mesecons_lightstone/locale/mesecons_lightstone.de.tr [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# textdomain: mesecons_lightstone
+Red Lightstone=Roter Leuchtstein
+Green Lightstone=Grüner Leuchtstein
+Blue Lightstone=Blauer Leuchtstein
+Grey Lightstone=Grauer Leuchtstein
+Dark Grey Lightstone=Dunkelgrauer Leuchtstein
+Yellow Lightstone=Gelber Leuchtstein
+Orange Lightstone=Orange Leuchtstein
```
Comment (Desour) [inline mesecons_lightstone/locale/mesecons_lightstone.de.tr:8]:
```suggestion
Orange Lightstone=Oranger Leuchtstein
```
oder
```suggestion
Orange Lightstone=Orangener Leuchtstein
```
idk

File: mesecons_materials/locale/mesecons_materials.de.tr [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# textdomain: mesecons_materials
+Glue=Klebstoff
+Fiber=Kabel
```
Comment (Desour) [inline mesecons_materials/locale/mesecons_materials.de.tr:3]: This is the insulator item.
Maybe "Gewebe"?

**Review by Desour — COMMENTED** [review]
TBH, I kinda hate in-game translations. x)
But that's not a reason to be against this.

**Comment by Wuzzy2** [comment]
“orange” is correct as the word “orange” has no flection in German. The same translation is used in MTG.

I’m not sure about the translation of "Fiber". What about "Faser”?

When I think about this, maybe the English word for this item is just bad because “Fiber” is an awfully generic word that could mean anything.

**Comment by fluxionary** [comment]
i'm only here incidentally, and just want to add some comments that i don't think need to be acted on. 

looking at the icon for the item called "fiber", it is a woven sheathing intended to provide insulation to a cable. the uses for the item validate this. the name "fiber" would be a [synecdoche](https://en.wikipedia.org/wiki/Synecdoche), referring to individual strands of material used to weave such a sheath. the resulting item is "a thing made of fiber". in english, it makes sense to an engineer or electrician when they see the icon, but not everyone has that sort of experience. on the other hand, not everyone can *see* the icon, though i've yet to meet a blind minetest player, so perhaps this isn't so important. 

using [beolingus](https://dict.tu-chemnitz.de/dings.cgi?lang=en&query=Fiber), Faser seems to be the most literal translation for the relevant meaning, though it looks like it's as non-specific in german as it is in english, and may introduce unintended ambiguities. should you be able to build unified inventory bags, or ship sails, out of this fiber, or just string and wool?

**Comment by Wuzzy2** [comment]
I changed the translation of "Fiber" from "Kabel" to "Faser".

**Comment by Wuzzy2** [comment]
Bump.

File: mesecons_delayer/locale/mesecons_delayer.de.tr [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# textdomain: mesecons_delayer
+Delayer=Verzögerer
+You hacker you=Sie Hacker!
```
Comment (sfan5) [inline mesecons_delayer/locale/mesecons_delayer.de.tr:3]: missing `!`?

File: mesecons_insulated/locale/mesecons_insulated.de.tr [hunk 5]
```diff
@@ -0,0 +1,2 @@
+# textdomain: mesecons_insulated
+Straight Insulated Mesecon=Isolierte Mesecongerade
```
Comment (sfan5) [inline mesecons_insulated/locale/mesecons_insulated.de.tr:2]: maybe it'd be better to change it to just "Isolated Mesecon"

File: mesecons_pistons/locale/mesecons_pistons.de.tr [hunk 6]
```diff
@@ -0,0 +1,7 @@
+# textdomain: mesecons_pistons
+Piston=Kolben
+Activated Piston Base=Aktivierter Kolbenkörper
+Piston Pusher=Kolbenschieber
+Sticky Piston=Klebriger Kolben
```
Comment (sfan5) [inline mesecons_pistons/locale/mesecons_pistons.de.tr:5]: vllt. "Haftender Kolben"

**Review by sfan5 — APPROVED** [review]
LGTM otherwise

**Comment by Wuzzy2** [comment]
* "You hacker you" had no "!" so I removed it from German now
* "Straight Insulated Mesecon": Sounds like you request a change to the source string which I'm obviously not going to do in a pure translation PR
* "Sticky Piston": done

Is this OK?

**Review by sfan5 — APPROVED** [review]

**Pull request merged**